Cargo.toml

```diff
@@ -121,6 +121,7 @@ gdk-pixbuf = "0.16"
 libappindicator = { version = "0.8.0", optional = true }
 dirs-next = { version = "2.0.0", optional = true }
 x11-dl = "2.21"
+zbus = "3.13"
```

Comment:
btw, I think the latest zbus release (3.13.1) will cause dependency conflict with `tauri:next` branch due to
https://github.com/dbus2/zbus/commit/dfbfd21ab42eb46d73d83e4ab6ec0ea86f957609 is not released yet.

ref: https://github.com/tauri-apps/tauri/pull/6845

Reply:
we've unlocked the winnow dependency in tauri again so it shouldn't be that much of an issue anymore, _but_ for this to really work we'll have to lower the minor version specified here or just go with `"3"`.

Reply:
Well, idk what to say

Reply:
Though at max I guess it will cause dependency cloning?

Reply:
> Though at max I guess it will cause dependency cloning?

Only if the major versions would be different. For 2x the same major version cargo will complain that it couldn't find a fitting version.

Reply:
Well, changing zbus to just `3` will help?

Reply:
yes

Reply:
✅

Reply:
oh okay, I see no problem.